src/cmd/generate.rs

```diff
@@ -1,11 +1,17 @@
 use clap::{Args, Subcommand};
+use ed25519_zebra::*;
+use rand::thread_rng;
+use colored::Colorize;
 
 #[derive(Args)]
 pub struct Generate {
     #[clap(subcommand)]
     command: GenerateCommands,
 }
 
+const DID_KEY_PREFIX:&str = "did:key:z";
+const DID_KEY_PREFIX_ED25519:[u8; 2] = [237, 1];
```

Comment:
Oh interesting! The existing did-key crates don't have Ed25519?

Reply:
The one did crate I saw had no documentation and looked unfinished, so I just made my own as it was pretty quick